tests/eval_tests/utils.py

```diff
@@ -82,6 +82,48 @@ def add_metric(
             error
         ])
 
+def add_client_tool_call_metric(
```

Comment:
This function looks to be identical to `add_metric` other than it tracks `expected_tool_call` instead of `server_type` is that right? If so, can you combine these into a single function?